rubric/rubric_peer-review.md

```diff
@@ -17,7 +17,7 @@ This rubric is for assessing one student's peer review of another student's work
 	<td>The review is so generic that it's hard to determine which assignment it's reviewing. It doesn't offer any praise and yet doesn't offer any suggestions either.</td>
 	<td></td>
```

Comment:
What about something along the lines of:

_The review identifies both positive and negative sides. However, suggestions for improvement are either not provided, or unfeasible._

for this grade range?